api/poetry.lock

```diff
@@ -501,20 +501,6 @@ category = "main"
 optional = false
 python-versions = ">=3.7"
 files = [
-    {file = "Pillow-9.4.0-1-cp310-cp310-macosx_10_10_x86_64.whl", hash = "sha256:1b4b4e9dda4f4e4c4e6896f93e84a8f0bcca3b059de9ddf67dac3c334b1195e1"},
```

Comment:
why is this file modify and not the `pyproject.toml` ?

Reply:
I updated the lock file. Everything's good now !